pkg/features/features.go

```diff
@@ -41,23 +41,27 @@ const (
 	// blockdevice UUID algorithm mentioned in
 	// https://github.com/openebs/openebs/pull/2666
 	GPTBasedUUID Feature = "GPTBasedUUID"
+	// GRPC feature flag starts the GRPC server which provides functionality to manage block devices
+	GRPC Feature = "GRPC"
```

Comment:
Better to use `GRPCServer` rather than just grpc.

Reply:
Done